en/docs/remote-apps/connect-remote-applications.md

```diff
@@ -111,7 +111,7 @@ Observability data is displayed as shown in the example below.
 
 !!! Tip
     Alternatively, you can navigate to the same page in the Choreo Console as follows:<br/><br/>
-    1. Sign in to the Choreo Console at [https://console.choreo.dev/](https://console.choreo.dev/).<br/><br/>   
+    1. Sign in to the Choreo Console at [https://console.choreo.dev/](https://console.choreo.dev/).<br/><br/>
```

Comment:
one <br> tag would be enough right?
This way there is an unnecessary new line isn't it?

Reply:
Made it one <br/> tag in each line.